apps/full-site-editing/package.json

```diff
@@ -45,8 +48,10 @@
 		"@wordpress/keycodes": "^2.4.0",
 		"@wordpress/nux": "^3.4.0",
 		"@wordpress/plugins": "^2.2.0",
+		"@wordpress/scripts": "^3.4.0",
```

Comment:
This and `enzyme` should be `devDependencies`, shouldn't they?

If they're `devDependencies`, they'd need to remove to the root package.json.

This relates to my comment here: https://github.com/Automattic/wp-calypso/pull/35729#issuecomment-524764943

Reply:
Thanks for pointing me at that. Much appreciated.

I realised last week that these run from the root and not the Plugin directory when I was trying to configure ESLint rules.

Reply:
@sirreal I've moved them to `devDependencies`. You seem to be implying these deps should be in the root `package.json`. I'm happy to do this if that gets this PR mergable. Looks like there is a wider discussion as to whether the apps should define their own deps (surely they should!?) so I don't want to get this PR blocked by that discussion if possible. 

Your advice on unblocking would be much appreciated.

Reply:
Bringing over conversation from another channel p1566901364144200-slack-team-calypso:

- `devDependencies` _must_ live at the root
- all of these are already covered except `@wordpress/scripts`
- We can add it, but it's a heavy package for limited use
- An alternative is to use `npx` to install on demand

I'd prefer using `npx` initially to install on demand for this package's scripts, it's easy to add the dependency later.

Reply:
We discussed this "offline" and decided `npx` is the best interim for `@wordpress/scripts`.

Something like

```
npx --package @wordpress/scripts wp-scripts test-unit-js
```

Reply:
Ok this has been implemented. The initial `npm run test:unit:watch` is now quite slow to boot up, but once it's going in `watch` mode it isn't a big impact.

Reply:
Coming full circle, we had some concerns around the monorepo approach and vision. The concerns are valid, but this app has already sidestepped them by operating as an independent app (cd to the app root, install/run/whatever).

That doesn't align with the monorepo vision, but we also don't have a good story for apps that Calypso does not depend on. `lerna bootstrap` and `yarn` workspaces are the sorts of tools we need for this case, but each have their drawbacks that we're not comfortable taking on at this time.

So, please proceed installing `devDependencies` in your app and I'll get out of your way 😉